daemon/api_model.go

```diff
@@ -44,13 +44,6 @@ var (
 	}
 )
 
-type assertType int
-
-const (
-	serialType assertType = iota
```

Comment:
Looks like this was introduced back when `snap model` got added. @anonymouse64 do you recall if there was an idea to make use of those enums?

Reply:
no idea, but if it's not needed anymore it was probably added as part of an initial implementation that got refactored away